libs/ui/src/components/atoms/tooltip/types.ts

```diff
@@ -0,0 +1,7 @@
+export type TooltipProps = {
+  content?: string | React.ReactNode;
```

Comment:
Nice. I love this flexibility -- that's how I've structured types for `content` before